style.css

```diff
@@ -0,0 +1,32 @@
+.orange-margin {
+  border-top: 8px solid #F5C85F;
+}
+
+* {
+  font-family: 'Open Sans', sans-serif;
+}
+
+.main-images {
+  display:flex;
+  flex-direction:row;
+  justify-content:center;
+  align-items:center;
+  background-color:white;
+  margin: 40px;
+  flex-wrap:wrap;
+}
+
+.vertical-line {
+  display:flex;
+  margin:10px;
+}
+
+.gradient {
+  height: 200px;
+  width:3px;
+  background:#FFD700;
+  background: linear-gradient(rgba(255, 215,0,0), rgba(255, 215,0,1), rgba(255, 215,0,0));
```

Comment:
`rgba(255, 215,0,0) `

when the last value is 0 the other values don't matter since the color is transparent. so you can just say `transparent` instead

`rgba(255, 215,0,1)`

when the last value is 1 you can (and should) write it in hex notation to be consistent